lib/crystalball.rb

```diff
@@ -10,6 +10,7 @@
 require 'crystalball/map_generator'
 require 'crystalball/map_generator/configuration'
 require 'crystalball/map_generator/coverage_strategy'
+require 'crystalball/rails/map_generator/action_view_strategy'
```

Comment:
this shouldn't be require by default. It should be required in `crystalball/rails` only